apps/home/models.py

```diff
@@ -124,13 +126,52 @@ def answers_dict(self):
 
     @property
     def answers_table(self):
-        data = self.answers_dict
-        output = "<table class='table table-bordered'>"
-        output += "<tr><th>Question</th><th>Answer</th></tr>"
-        for k, v in data.items():
-            output += f"<tr><td><b>{k}</b></td><td>{v}</td></tr>"
+        answers = self.answers_dict
+        comments = self.comments_dict
+        grades = self.grades_dict
+        
+        output = "<form id='bulk-comments-form' method='post'>"
+        output += "<table class='table table-bordered'>"
+        output += "<tr><th>Question</th><th>Answer</th><th>Comment</th><th>Grade</th></tr>"
+        for answer_name, answer_value in answers.items():
+            comment = comments.get(answer_name, {}) if comments else {}
+            grade = grades.get(answer_name, {}) if grades else {}
+
+            comment_value = comment.get("comment", "") if comment else ""
+            grade_value = grade.get("grade", "") if (grade or grade == 0) else ""
+
+            output += f"""
+            <tr>
+                <td><b>{answer_name}</b></td>
+                <td>{answer_value}</td>
+                <input type="hidden" name="name[]" value="{answer_name}">
+                <input type="hidden" name="user[]" value="{self.user_id}">
+                <td>
+                    <textarea name="comment[]" class="form-control" placeholder="Comentário">{comment_value}</textarea>
+                </td>
+                <td>
+                    <input type="number" min="0" max="100" name="grade[]" class="form-control" placeholder="Nota" value="{grade_value}">
+                </td>
+            </tr>
+            """
         output += "</table>"
+        output += "<div id='bulk-comment-result' class='mt-2'></div>"
+        output += "</form>"
         return output
+    
+    @property
+    def comments_dict(self):
+        try:
+            return json.loads(self.comments) if self.comments else {}
```

Comment:
is the if-else statement here really necessary? Check this out:
```
>>> import json
>>> json.loads("")
Traceback (most recent call last):
...
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
>>> json.loads(None)
Traceback (most recent call last):
...
TypeError: the JSON object must be str, bytes or bytearray, not NoneType
```

i.e., the try-except will already do the job of handling possible non-expected values there, no?